base/commandLine.cpp

```diff
@@ -55,22 +55,15 @@ namespace Base {
 
 #ifndef DISABLE_COMMAND_LINE
 
-#if !defined(__DS__)
+#ifndef DISABLE_HELP_STRINGS
 static const char USAGE_STRING[] =
 	"%s: %s\n"
 	"Usage: %s [OPTIONS]... [GAME]\n"
 	"\n"
 	"Try '%s --help' for more options.\n"
 ;
-#endif
 
 // DONT FIXME: DO NOT ORDER ALPHABETICALLY, THIS IS ORDERED BY IMPORTANCE/CATEGORY! :)
-#if defined(ANDROID) || defined(__DS__) || defined(__3DS__)
```

Comment:
And what about Android? You removed it here but I do not see it added in the `configure`

Reply:
I explained it in the commit message.
The ANDROID define was not a thing since 8de5edde9599b6ca00c23e753c00676d7bc9b335.
So the `#if` was not taken anymore for 10 years.
I just checked and, as of today, `libscummvm.so` does contain the help strings.